packages/wranglerjs-compat-webpack-plugin/src/index.ts

```diff
@@ -194,7 +195,7 @@ export class WranglerJsCompatWebpackPlugin extends Plugin {
       console.warn(
         `Installing deps in ${this.packageDir}, but you should do this yourself...`
       );
-      await execa("npm", ["install"], {
+      child_process.spawnSync("npm", ["install"], {
```

Comment:
why not  `execaSync`?

Reply:
it causes problems with webpack 4, since execa is an ESModule and webpack.config.js needs any plugins to be CommonJS Modules